stdlib/asyncio/base_events.pyi

```diff
@@ -64,11 +64,7 @@ class Server(AbstractServer):
 
 class BaseEventLoop(AbstractEventLoop):
     def run_forever(self) -> None: ...
-    # Can't use a union, see mypy issue  # 1873.
```

Comment:
This comment was just copied from the existing comment in `events.pyi`, as part of https://github.com/python/typeshed/pull/2960